qa-core/src/config/internal-api/TestConfiguration/applications.ts

```diff
@@ -32,13 +33,21 @@ export default class AppApi {
 
   async canRender(): Promise<[Response, boolean]> {
     const response = await this.api.get("/routing/client")
+    expect(response).toHaveStatusCode(200)
     const json = await response.json()
     const publishedAppRenders = Object.keys(json.routes).length > 0
-    expect(response).toHaveStatusCode(200)
     expect(publishedAppRenders).toBe(true)
     return [response, publishedAppRenders]
   }
 
+  async failRender(): Promise<[Response, responseMessage]> {
```

Comment:
Doesn't seem like this is called anywhere

Reply:
Thank you for catching that.